### DISCUSSION TITLE
How to get syntax highlighting working

### DISCUSSION THREAD

**Discussion by vivekragunathan** [thuliteio/doks#693] [Q&A]
Initially, there were some languages that were syntax highlighted (bash for one) but not others. After searching for a bit, I found [**this**](https://github.com/h-enk/doks/discussions/401#discussioncomment-1047945) comment and also this [**doc**](https://getdoks.org/docs/extensions/highlight-js/#3-configure) on how to set up highlight.js for syntax highlighting. Followed the suggestion and get a spew of errors in the console but syntax not highlighted. 

Is there anything guidance on how to get it working?

**Comment by vivekragunathan** [comment]
@h-enk Nudging for visibility

**Reply by h-enk** [reply]
What does your `./assets/js/highlight.js` look like + what errors are you getting?

**Comment by imroc** [comment]
I also follow this [doc](https://getdoks.org/docs/extensions/highlight-js/) to setup syntax highlighting, but it did not work:
1. set `highLight = false` in `params.toml`.
2. set `@import "highlight.js/scss/monokai"` in `app.scss`;

**Reply by roheclassic** [reply]
you don't need to change the settings in params.toml from the default.
All you have to do is the second (in your comment) only.

OK: ```highLight = true``` in params.toml

**Reply by imroc** [reply]
Misled by the docs, I figured it out myself. The website is online: https://req.cool (use monokai)

**Reply by roheclassic** [reply]
Actually,I was hesitant to answer this question because I thought maybe you solved it yourself.

Bytheway,your own product req is nice.
The code with it looks very concise and elegant.
(I like chainable methods.)

I'm not familiar with go, but I'll try to use it for work.

**Reply by imroc** [reply]
thanks 😉
